corehq/apps/case_search/utils.py

```diff
@@ -119,3 +122,72 @@ def _add_case_property_queries(self):
                 self.search_es = self.search_es.xpath_query(self.domain, query)
             else:
                 self.search_es = self.search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
+
+
+def get_related_cases(cases, app_id):
+    """
+    Fetch related cases that are necessary to display any related-case
+    properties in the app requesting this case search.
+
+    Returns list of CommCareCase objects for adding to CaseDBFixture.
+    """
+    if not cases:
+        return []
+
+    app = get_app_cached(cases[0].domain, app_id)
+    paths = get_related_case_relationships(app, cases[0].type)
```

Comment:
I realize it's redundant but I think it would be better to pass the domain and case type in as function arguments rather than deducing them from the first case in the list. It just makes it more explicit.

Reply:
Ha, I was debating that while writing this. Fixed in a9a2ffbb031